!article/article.md

```diff
@@ -0,0 +1,170 @@
+Pinned-Dependencies | Does the project declare and pin dependencies? | Medium | **Yes**
+Packaging | Does the project build and publish official packages from CI/CD, e.g. GitHub Publishing | Medium | **Yes**
+SAST | Does the project use static code analysis tools, e.g. CodeQL, LGTM (deprecated), SonarCloud? | Medium | No
+Security-Policy | Does the project contain a security policy? | Medium | No
+Signed-Releases | Does the project cryptographically sign releases? | High | No
+Token-Permissions | Does the project declare GitHub workflow tokens as read only? | High | No
+Vulnerabilities | Does the project have unfixed vulnerabilities? Uses the OSV service. | High | No
+Webhooks | Does the webhook defined in the repository have a token configured to authenticate the origins of requests? | Critical | No
+
+
+# What is supported for .NET/NuGet
+For the past two years, Scorecard has started to implement dedicated support for projects within the .NET ecosystem, which use the NuGet package manager. The most recent features added are support for checking pinned dependencies when restoring packages using a lock file and when using Central Package Management.
+
+## Running Scorecard on a NuGet package
+You can run Scorecard with the repository name as the input (`--repo=[YOUR REPO NAME]`) or you can run it with the name of a NuGet package for one that is hosted on Nuget.org. To do that you need to specify the `--nuget` flag and provide the package name for which you want the corresponding GitHub source code to be checked.
+
+For example: `--nuget=Newtonsoft.Json`
+
+Note that this feature is only supported for packages that define the repository attribute in their  package metadata .nuspec file.
+
+## Packaging 
+This check evaluates whether the project is published as a package. Packages make it easier for users to download, install, update, and receive security patches via a package manager. The check currently looks for GitHub packaging workflows and language-specific GitHub Actions that upload the package to NuGet and other package managers. 
+
+Note that if a project uses different packaging tools, it may still receive a low score, as Scorecard can't detect all packaging methods. A low score doesn’t necessarily mean the project is at risk. If your packaging method isn’t recognized, consider opening an issue with the Scorecard maintainers. 
+
+In case your project receives a low score on this check, you can try the following remediation steps:
+
+- Publish your project as a downloadable package, e.g., if hosted on GitHub, use GitHub's mechanisms for publishing a package.
+- If hosted on GitHub, use a GitHub action to release your package to language-specific hubs.
+
+## Pinned dependencies 
+For restoring NuGet packages, it is considered best practice to use Central Package Management (CPM) and/or a lock file to fully lock down the package dependency graph, ensuring consistent and repeatable builds while preventing “it works on my machine" issues.
```

Comment:
perhaps more importantly than preventing "works on my machine" issues is [counter dependency confusion (aka substitution) attacks](https://azure.microsoft.com/en-us/resources/3-ways-to-mitigate-risk-using-private-package-feeds/). wdyt?

Reply:
For some reason, I cannot open the link, it redirects me to the homepage.